lms/djangoapps/teams/static/teams/js/views/topic_header.js

```diff
@@ -0,0 +1,36 @@
+;(function (define) {
+    'use strict';
+    define([
+        'backbone',
+        'underscore',
+        'gettext',
+        'text!teams/templates/topic-header.underscore'
+    ], function (Backbone, _, gettext, headerTemplate) {
+        var TopicHeader = Backbone.View.extend({
+            initialize: function (options) {
+                this.collections = options.collection;
+                this.collection.bind('add', _.bind(this.render, this));
+                this.collection.bind('remove', _.bind(this.render, this));
+                this.collection.bind('reset', _.bind(this.render, this));
+            },
+
+            render: function () {
+                var currentlyViewing,
+                    start = this.collection.start,
+                    perPage = this.collection.perPage,
+                    end = start + perPage,
+                    size = this.collection.totalCount;
+                // Translators: becomes either "all 10 topics" or "1 through 5 of 10 topics"
```

Comment:
I realize this PR is still in-progress, but these translators comments won't be seen by translators since they must be on the line directly above a call to `gettext`. Moreover, instead of appending strings directly, we should be writing code like `gettext(interpolate('i have %(num_things)s things', {num_things: 5}, true));` so that translators have context for what they're translating. In this case, they'll only get `" through "` and `" of "`.